components/stacks-network/src/lib.rs

```diff
@@ -86,15 +94,30 @@ pub async fn do_run_devnet(
         .with_writer(non_blocking)
         .try_init();
 
-    let ip_address_setup = devnet.prepare_network().await?;
+    let ip_address_setup = {
+        let hosts = ServicesMapHosts {
+            bitcoin_node_host: format!(
+                "{BITCOIND_CHAIN_COORDINATOR_SERVICE}.{namespace}.svc.cluster.local:{BITCOIND_RPC_PORT}"
+            ),
+            stacks_node_host: format!("{STACKS_NODE_SERVICE}.{namespace}.svc.cluster.local:{STACKS_NODE_RPC_PORT}"),
+            postgres_host: format!("{STACKS_API_SERVICE}.{namespace}.svc.cluster.local:{STACKS_API_POSTGRES_PORT}"),
+            stacks_api_host: format!("{STACKS_API_SERVICE}.{namespace}.svc.cluster.local:{STACKS_API_PORT}"),
+            stacks_explorer_host: "localhost".into(), // todo (micaiah)
+            bitcoin_explorer_host: "localhost".into(), // todo (micaiah)
+            subnet_node_host: "localhost".into(), // todo (micaiah)
+            subnet_api_host: "localhost".into(), // todo (micaiah)
+        };
+        devnet.set_services_map_hosts(hosts.clone());
+        hosts
+    };
 
+    println!("bitcoind host {}", ip_address_setup.bitcoin_node_host);
```

Comment:
Could we use the logger coming from `ctx`, instead of `printf`?

Reply:
That print statement shouldn't have been there anyways, thanks!